### PR TITLE
add mapbox-vector-tile

### PR DESCRIPTION
**Pull request by rluria14**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @benhuff will you be maintainers?

**Comment by benhuff** [comment]
@rluria14 yes, i will be a maintainer.

**Comment by adbeda** [comment]
yes

File: recipes/mapbox-vector-tile/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - pyclipper
+    - python
+    - setuptools
+    - shapely
+
+test:
+  imports:
+    - mapbox_vector_tile
+    - mapbox_vector_tile.Mapbox
+    - tests
```
Comment (chrisburr) [inline recipes/mapbox-vector-tile/meta.yaml]: This import shouldn't be provided by the package else would have conflicts with other packages.

The `exclude` argument to `find_packages` is probably missing upstream in `setup.py`. Can you patch it?
```python
find_packages(exclude=['tests'])
```

Comment (rluria14) [reply recipes/mapbox-vector-tile/meta.yaml]: Patch has been added. Thanks.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**